axelrod/interaction_utils.py

```diff
@@ -87,6 +88,32 @@ def compute_normalised_cooperation(interactions):
     return normalised_cooperation
 
 
+def compute_state_distribution(interactions):
+    """
+    Returns the count of each state for a set of interactions.
+    """
+    if len(interactions) == 0:
```

Comment:
You don't need to change them all (unless you wanted to!) but if you could keep the new ones according the convention that would be much appreciated. We have #347 as an ongoing issue to back-fix the formatting.